src/Functions/stl.hpp

```diff
@@ -0,0 +1,510 @@
+/*!
```

Comment:
Future readers will wonder where this file comes from, what it does, if it is part of ClickHouse and if it isn't, what are the changes to the original version.

Therefore, let's add a comment:
```cpp
/// Dump of https://github.com/ankane/stl-cpp/blob/f3e1cd02b4f0341a7539ed1a171b21b14b37ed51/include/stl.hpp. Added to ClickHouse source code and not referenced as a submodule because of <Reason 1> and <Reason 2>.
```

(please check that the link points to the version you used and fill in the reasons, thanks!)